### ISSUE TITLE
automatically accept GitHub invitations to organizations

### ISSUE THREAD

**Issue by yegor256** [yegor256/0pdd#143]
At the moment we only accept repository invitations. Let's accept ones into organizations.

**Comment by 0crat** [comment]
Job `gh:yegor256/0pdd#143` is in scope.

**Comment by 0crat** [comment]
Bug was reported: +15 points just awarded to @yegor256, total is [+5290](http://www.0crat.com/u/yegor256).

**Comment by 0crat** [comment]
Job `gh:yegor256/0pdd#143` assigned to @tedtoer, please go ahead ([policy](http://datum.zerocracy.com/pages/policy.html)).

**Comment by 0crat** [comment]
@tedtoer this job was assigned to you 5 days ago. It will be taken away from you soon, unless you close it, see [par.8](http://datum.zerocracy.com/pages/policy.html#8).

**Comment by 0crat** [comment]
@tedtoer this job was assigned to you 8 days ago. It will be taken away from you soon, unless you close it, see [par.8](http://datum.zerocracy.com/pages/policy.html#8).

**Comment by 0crat** [comment]
@tedtoer resigned from `gh:yegor256/0pdd#143`, please stop working.

**Comment by 0crat** [comment]
Job `gh:yegor256/0pdd#143` assigned to @tedtoer, please go ahead ([policy](http://datum.zerocracy.com/pages/policy.html)).

**Comment by 0crat** [comment]
@tedtoer resigned from `gh:yegor256/0pdd#143`, please stop working.

**Comment by 0crat** [comment]
There is an unrecoverable failure on my side. Please, submit it [here](https://github.com/zerocracy/datum):

```
java.lang.IllegalArgumentException: Can't process this type of notification: {"title":"#354 ignore @0crat assignments","url":"https://api.github.com/repos/zerocracy/farm/commits/97096402420866657c759a460ed0c282bcb3a555","latest_comment_url":"https://api.github.com/repos/zerocracy/farm/commits/97096402420866657c759a460ed0c282bcb3a555","type":"Commit"}
	at com.zerocracy.radars.github.ReOnComment.react(ReOnComment.java:103)
	at com.zerocracy.radars.github.ReOnReason.react(ReOnReason.java:56)
	at com.zerocracy.radars.github.Reaction$Chain.react(Reaction.java:74)
	at com.zerocracy.radars.github.ReLogged.react(ReLogged.java:55)
	at com.zerocracy.radars.github.RbOnComment.react(RbOnComment.java:60)
	at com.zerocracy.radars.github.RbByActions.react(RbByActions.java:65)
	at com.zerocracy.radars.github.Rebound$Chain.react(Rebound.java:74)
	at com.zerocracy.radars.github.RbSafe.lambda$react$0(RbSafe.java:65)
	at org.cactoos.func.FuncWithFallback.apply(FuncWithFallback.java:299)
	at org.cactoos...
```

[1.0-SNAPSHOT](https://github.com/zerocracy/farm/tree/1.0-SNAPSHOT) /cc @yegor256

**Comment by 0crat** [comment]
There is an unrecoverable failure on my side. Please, submit it [here](https://github.com/zerocracy/datum):

```
java.lang.IllegalArgumentException: Can't process this type of notification: {"title":"#354 ignore @0crat assignments","url":"https://api.github.com/repos/zerocracy/farm/commits/97096402420866657c759a460ed0c282bcb3a555","latest_comment_url":"https://api.github.com/repos/zerocracy/farm/commits/97096402420866657c759a460ed0c282bcb3a555","type":"Commit"}
	at com.zerocracy.radars.github.ReOnComment.react(ReOnComment.java:103)
	at com.zerocracy.radars.github.ReOnReason.react(ReOnReason.java:56)
	at com.zerocracy.radars.github.Reaction$Chain.react(Reaction.java:74)
	at com.zerocracy.radars.github.ReLogged.react(ReLogged.java:55)
	at com.zerocracy.radars.github.RbOnComment.react(RbOnComment.java:60)
	at com.zerocracy.radars.github.RbByActions.react(RbByActions.java:65)
	at com.zerocracy.radars.github.Rebound$Chain.react(Rebound.java:74)
	at com.zerocracy.radars.github.RbSafe.lambda$react$0(RbSafe.java:65)
	at org.cactoos.func.FuncWithFallback.apply(FuncWithFallback.java:299)
	at org.cactoos...
```

[1.0-SNAPSHOT](https://github.com/zerocracy/farm/tree/1.0-SNAPSHOT) /cc @yegor256

**Comment by 0crat** [comment]
Job `gh:yegor256/0pdd#143` assigned to @tedtoer, please go ahead ([policy](http://datum.zerocracy.com/pages/policy.html)).

**Comment by 0crat** [comment]
The user @prondzyn[/z](https://www.0crat.com/u/prondzyn) resigned from [#143](https://github.com/yegor256/0pdd/issues/143), please stop working

**Comment by 0crat** [comment]
Resigned on delay, see [§8](http://www.zerocracy.com/policy.html#8): -30 points just awarded to @prondzyn[/z](https://www.0crat.com/u/prondzyn), total is [-90](https://www.0crat.com/u/prondzyn)

**Comment by SilasReinagel** [comment]
@yegor256 Where does the code that accepts repository invites?

**Comment by yegor256** [comment]
@SilasReinagel have you seen this? http://www.yegor256.com/2015/01/15/how-to-cut-corners.html What do we do when something is not clear? :)

**Comment by yegor256** [comment]
@SilasReinagel the code is here: https://github.com/yegor256/0pdd/blob/master/0pdd.rb#L240-L269

**Comment by 0crat** [comment]
@silasreinagel[/z](https://www.0crat.com/u/silasreinagel) this job was assigned to you 5days ago. It will be taken away from you soon, unless you close it, see [§8](http://www.zerocracy.com/policy.html#8). Read [this](http://www.yegor256.com/2014/04/13/no-obligations-principle.html) and [this](http://www.yegor256.com/2014/11/24/principles-of-bug-tracking.html), please.

**Comment by SilasReinagel** [comment]
@0crat wait on PR merge

**Comment by 0crat** [comment]
> @0crat wait on PR merge ([here](https://github.com/yegor256/0pdd/issues/143#issuecomment-381819681))

@SilasReinagel The impediment for [#143](https://github.com/yegor256/0pdd/issues/143) was registered successfully by @silasreinagel[/z](https://www.0crat.com/u/silasreinagel)

**Comment by 0crat** [comment]
@rultor[/z](https://www.0crat.com/u/rultor) the issue is closed not by @yegor256[/z](https://www.0crat.com/u/yegor256) (its creator); I won't close the order; please, re-open it and ask @yegor256[/z](https://www.0crat.com/u/yegor256) to close it

**Comment by SilasReinagel** [comment]
@yegor256 Apparently we need to do this on every ticket. Can you close and reopen this ticket?

**Comment by SilasReinagel** [comment]
@yegor256 Can you close and reopen this ticket? It's still open in 0crat.

**Comment by 0crat** [comment]
Order was finished: +30 point(s) just awarded to @silasreinagel[/z](https://www.0crat.com/u/silasreinagel)

**Comment by 0crat** [comment]
The job [#143](https://github.com/yegor256/0pdd/issues/143) is now out of scope
